doc/administrator/tinyows.rst

```diff
@@ -2,46 +2,24 @@
 
 WFS-T with TinyOWS
 ==================
-Based on `TinyOWS <http://mapserver.org/tinyows/>`__ c2cgeoportal layers can be
+
+Based on `TinyOWS <http://mapserver.org/tinyows/>`__ c2cgeoportal mapserver layers can be
```

Comment:
Why mapserver layers, it also for qgisserver layers AFAIK?

Reply:
It can be but QGIS server directly support WFS-T, I will add a note about that :-)